cmake/Platform/Linux/Packaging_linux.cmake

```diff
@@ -69,4 +69,24 @@ elseif("$ENV{O3DE_PACKAGE_TYPE}" STREQUAL "DEB")
         ${CMAKE_BINARY_DIR}/cmake/Platform/Linux/Packaging/postrm
     )
 
+    # Scripts to add executable files to PATH
+    configure_file("${CMAKE_CURRENT_LIST_DIR}/Packaging/o3de.in"
+        ${CMAKE_CURRENT_LIST_DIR}/Packaging/o3de
+        @ONLY
+    )
+    configure_file("${CMAKE_CURRENT_LIST_DIR}/Packaging/o3de.assetprocessor.in"
+        ${CMAKE_CURRENT_LIST_DIR}/Packaging/o3de.assetprocessor
+        @ONLY
+    )
+    configure_file("${CMAKE_CURRENT_LIST_DIR}/Packaging/o3de.editor.in"
+        ${CMAKE_CURRENT_LIST_DIR}/Packaging/o3de.editor
+        @ONLY
+    )
+    ly_install_files(PROGRAMS FILES
+        ${CMAKE_CURRENT_LIST_DIR}/Packaging/o3de
+        ${CMAKE_CURRENT_LIST_DIR}/Packaging/o3de.assetprocessor
+        ${CMAKE_CURRENT_LIST_DIR}/Packaging/o3de.editor
+        DESTINATION /usr/bin
+    )
```

Comment:
The generation of files should not be written to `${CMAKE_CURRENT_LIST_DIR}` as this modifies the user source tree.

Instead, these files should be written to the `${CMAKE_BINARY_DIR}` which corresponds to the user build directory.

Reply:
Thanks for the review, should be fixed in 195aeaf

Reply:
Thanks